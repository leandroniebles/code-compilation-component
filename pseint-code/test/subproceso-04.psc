SubProceso Booga(a,b)
  Escribir A
  Escribir B
FinSubproceso

Proceso PruebaFuncion
  Booga("Hola");
  Booga("Hola","Mundo","Cruel");
  Booga;
FinProceso

  
