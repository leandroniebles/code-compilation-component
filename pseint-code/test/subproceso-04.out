=== Line 7: SynError 268
=== Line 8: SynError 268
=== Line 9: SynError 264
